profiles/sftp_storage/README.md

```diff
@@ -0,0 +1,13 @@
+# sftp_storage
+
+## Usage
+
+*Add a description detailing what this profile does.*
```

Comment:
These lines with the asterisks are instructions for the profile writer, they should be removed after following them.